metadata/pkg/constants/Constants.go

```diff
@@ -0,0 +1,36 @@
+package constants
+
+const (
+	CACHE_NAME = "cachename"
+
```

Comment:
you may remove gap of line between two lines.

Reply:
done